modules/ROOT/pages/contribute.adoc

```diff
@@ -15,4 +15,7 @@ If you aren't already logged in to your GitHub account, you can do so using a li
 
 image:api-notebook1.png[User interface for running the API notebook to become a contributor]
 
+A Salesforce bot asks you in your pull request to complete this process if you submit the pull request without running the API Notebook first.
```

Comment:
My suggestions: 

I like this flipped around, as shown below, because if the user did run the API notebook, he/she  doesn't need to read what happens when it is not run.

This is a case where I would use future tense. I don't think that the user cares whether it is a bot that asks them this. All the user cares about is the returned message.

"If you submit the pull request without running the API Notebook first, you will receive a message in the pull request asking you to do so."

Reply:
brilliant! will do!